tests/sync_diff_inspector/snapshot/run.sh

```diff
@@ -20,15 +20,28 @@ check_contains "sourceDB don't equal targetDB" $OUT_DIR/snapshot_diff.log
 # fix.sql will be empty after check below, so backup it
 cp $OUT_DIR/fix.sql $OUT_DIR/fix.sql.bak
 
-echo "use snapshot compare data, data should be equal"
+echo "use snapshot compare data, test sql mode by the way, will return error, diff should not passed"
+mysql -uroot -h 127.0.0.1 -P 4000 -e "SET GLOBAL sql_mode = 'ANSI_QUOTES';"
+sleep 10
+mysql -uroot -h 127.0.0.1 -P 4000 -e "show variables like '%sql_mode%'"
+mysql -uroot -h 127.0.0.1 -P 4000 -e "show create table diff_test.test"
 cp config_base.toml config.toml
 echo "snapshot = \"$ts\"" >> config.toml
+sync_diff_inspector --config=./config.toml > $OUT_DIR/snapshot_diff.log || true
```

Comment:
Is `|| true` necessary?

Reply:
yes, diff will get error, and this script will exit